docs/pages/modules/autolinking.mdx

```diff
@@ -0,0 +1,196 @@
+installing a library would require configuring even up to three different package managers!
+
+Expo Autolinking is a mechanism that automates this process and narrows down the library installation process to the minimum — usually just installing the package from `npm` and re-running `pod install`.
+The core implementation can be found in the [`expo-modules-autolinking`](https://github.com/expo/expo/tree/main/packages/expo-modules-autolinking) package and is divided into three parts:
+- common JavaScript CLI tool with the module resolution algorithm
+- code that integrates with the Gradle build system for Android platform
+- Ruby script that integrates with CocoaPods for iOS platform
+
+## CLI Commands
+
+<APIBox>
+
+### `search`
+
+Searching is the first phase of resolving the Expo modules installed in the project. Its implementation is common for all platforms and is intended to
+just find which modules should be taken as Expo modules and figure out which version is of the highest precedence (in case of duplicates).
+
+<Terminal cmd={['npx expo-modules-autolinking search']} />
+
+Returns an object in JSON format with modules that have been found as dependencies:
+
+```json
+{
+  "expo-random": {
+    "path": "/absolute/path/to/node_modules/expo-random",
+    "version": "13.0.0",
+    "config": { // Contents of `expo-module.config.json`
+      "platforms": [ "ios", "android" ],
+      "ios": { "modules": [ "RandomModule" ] },
+      "android": { "modules": [ "expo.modules.random.RandomModule" ] }
+    },
+    "duplicates": [] // An array of other revisions (with lower precedence) of the same module
+  },
+  // more modules...
+}
+```
+
+</APIBox>
+<APIBox>
+
+### `resolve`
+
+Resolving is the second phase that is based on the results from the `search` command. It resolves each search result to an object with more (platform-specific) details,
+such as the path to the podspec or **build.gradle** files and module classes to link.
+
+<Terminal cmd={['npx expo-modules-autolinking resolve --platform <ios|android>']} />
+
+For example, with `--platform ios` option it returns an object in JSON format with an array of modules and resolved details for iOS platform:
+
+```json
+{
+  "modules": [
+    {
+      "packageName": "expo-random",
+      "packageVersion": "13.0.0",
+      "pods": [
+        {
+          "podName": "ExpoRandom",
+          "podspecDir": "/absolute/path/to/node_modules/expo-random/ios"
+        }
+      ],
+      "swiftModuleNames": [ "ExpoRandom" ],
+      "modules": [ "RandomModule" ],
+      "appDelegateSubscribers": [],
+      "reactDelegateHandlers": [],
+      "debugOnly": false
+    },
+    // more modules...
+  ]
+}
+```
+
+</APIBox>
+<APIBox>
+
+### `verify`
+
+Verifies the search results by checking whether there are no multiple revisions (duplicates) of the same package, otherwise an appropriate warning is shown.
+
+<Terminal cmd={['npx expo-modules-autolinking verify']} />
```

Comment:
Maybe we can add `$` to all commands, directly in the Terminal component?

Reply:
Yeah, that sounds good. I'm not sure, why it isn't and will have to check.

Reply:
This is a formatting sign for a line, and there are cases where we need to add the command output or other non-formatted value, and in cases like this we use lines without any formatting char at the beginning. 

Maybe this can be flipped around or the default can change, but this will require correcting manually at least a several pages.

Reply:
Thanks for your input here @Simek. Will create a task to change it and test the pages manually.